discordmenu/emoji/emoji_cache.py

```diff
@@ -54,7 +54,7 @@ def _emoji(e: Emoji) -> Emoji:
     def _emoji_name(e: Emoji) -> str:
         return e.name
 
-    def get_emoji(self, name: str) -> str:
+    def get_emoji(self, name: str, *, default: Optional[str] = None) -> str:
```

Comment:
this should instead make `name` take a list of strings and then return the emoji for the first valid one it finds